piet-coregraphics/src/lib.rs

```diff
@@ -336,9 +361,11 @@ impl<'a> RenderContext for CoreGraphicsContext<'a> {
         dst_rect: impl Into<Rect>,
         _interp: InterpolationMode,
     ) {
-        if let Some(cropped) = image.cropped(to_cgrect(src_rect)) {
-            // TODO: apply interpolation mode
-            self.ctx.draw_image(to_cgrect(dst_rect), &cropped);
+        if let CoreGraphicsImage::NonEmpty(image) = image {
+            if let Some(cropped) = image.cropped(to_cgrect(src_rect)) {
+                // TODO: apply interpolation mode
```

Comment:
Out of scope for this PR, which is fine, but this TODO bothers me. There should be a common impl with this and `draw_image` that takes care of interpolation mode.